.prettierrc.json

```diff
@@ -0,0 +1,11 @@
+{
+  "singleQuote": true,
+  "trailingComma": "all",
+  "useTabs": true,
```

Comment:
Is this tabs instead of spaces? I think that's going to mess some things up as the current rule is 2 spaces for indenting...

Reply:
Ooh - using tabs _will_ cause problems... can we stick to spaces, please?